client/i18n/locales/english/intro.json

```diff
@@ -1700,9 +1700,12 @@
     "title": "Front End Development",
     "intro": ["placeholder"],
     "blocks": {
-      "learn-basic-html-by-building-a-recipe-page": {
-        "title": "Learn Basic HTML by Building a Recipe Page",
-        "intro": ["", ""]
+      "workshop-cat-photo-app": {
+        "title": "Basic HTML Workshop - Build a Cat Photo App",
```

Comment:
I don't think we should have this as the title. 

We intend to have something in the UI to show users that this is in the Basic HTML module, and a label for the workshop. So I think we should just go with Build a Cat Photo App, or Learn (something) by Building a Cat Photo App.

Reply:
Ok, Let's go with  Build a Cat Photo App 👍